frontend/tests/features/cellGuide/cellGuide.test.ts

```diff
@@ -321,12 +321,18 @@ describe("Cell Guide", () => {
             expect(rowCountBefore).toBeGreaterThan(1);
 
             const dropdown = page.getByTestId(
-              CELL_GUIDE_CARD_MARKER_GENES_TABLE_DROPDOWN_ORGAN
+              CELL_GUIDE_CARD_GLOBAL_TISSUE_FILTER_DROPDOWN
             );
             await waitForElementAndClick(dropdown);
             await dropdown.press("ArrowDown");
             await dropdown.press("ArrowDown");
-            await dropdown.press("ArrowDown"); // selects kidney
+            await dropdown.press("ArrowDown");
```

Comment:
this is brittle, since if the option order changes, this test will break. Better to look for `li` element with `Brain` text!